src/Uno.UI/UI/Xaml/Style/Generic/SystemResources.xaml

```diff
@@ -3958,5 +3958,8 @@
 	<x:Double x:Key="TextBoxLeftHeaderMaxWidth">296</x:Double>
 	<x:Double x:Key="TimePickerLeftHeaderMaxWidth">296</x:Double>
 	<x:Double x:Key="ToggleSwitchLeftHeaderMaxWidth">296</x:Double>
-	
+
+	<!-- These resources don't exist on UWP, we add them to support using the WinUI ColorPicker style when the rest of the Fluent style package isn't present -->
+	<x:Double x:Key="SliderHorizontalThumbHeight">24</x:Double>
+	<x:Double x:Key="SliderHorizontalThumbWidth">8</x:Double>	
```

Comment:
The ColorPicker exists in the Windows.UI.Xaml namespace in Windows as well. I guess an alternative 'more correct' approach would be to add that old style to Uno.

I've kind of talked myself out of this already since: (1) The existing Windows template (not part of WinUI's fluent) isn't MIT licensed so would need to be partially re-developed. (2) This is really only a temporary work-around for a year or so until WinUI 3 becomes standard for all applications and MUX replaces WUX. (3) Uno doesn't plan to include the WUX version of this control anyway.